hr_utilization_report/__manifest__.py

```diff
@@ -0,0 +1,24 @@
+# Copyright 2018 Brainbean Apps (https://brainbeanapps.com)
+# License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl.html).
+
+{
+    'name': 'Task Logs Utilization Report',
+    'version': '12.0.1.0.0',
+    'category': 'Human Resources',
+    'website': 'https://github.com/OCA/hr-timesheet',
+    'author':
+        'Brainbean Apps, '
+        'Odoo Community Association (OCA)',
+    'license': 'AGPL-3',
+    'installable': True,
+    'application': False,
+    'summary': 'Generate Utilization Report from Task Logs',
+    'depends': [
+        'hr_timesheet',
+        'report_xlsx',
+    ],
+    'data': [
+        'report/hr_utilization_report.xml',
+        'wizards/hr_utilization_report_wizard.xml',
```

Comment:
`wizards` according to https://github.com/OCA/maintainer-tools/tree/master/template/module/wizards

Reply:
@alexey-pelykh It's Simple if file then `wizard`, If many then `wizards`

Reply:
@nikul-serpentcs not sure if that's correct, since that would mean that same approach (singular vs plural) would've been applied to models, tests, and views - but it's not the case.

@pedrobaeza what's the truth here?

Reply:
IMO This should be wizard always.

Reply:
Guidelines are for consistency, and you can see in ours that it should be plural:

https://github.com/OCA/odoo-community.org/blob/master/website/Contribution/CONTRIBUTING.rst#14directories

Odoo is not consistent in this thing (as well as in others), but they are improving.